packages/lib/src/components/GooglePay/types.ts

```diff
@@ -58,6 +58,14 @@ export interface GooglePayProps extends UIElementProps {
     allowedAuthMethods?: google.payments.api.CardAuthMethod[];
     allowedCardNetworks?: google.payments.api.CardNetwork[];
 
+    /**
+     * Set to true to request assuranceDetails. This object provides information
+     * about the validation performed on the returned payment data.
+     *
+     * @defaultValue false
+     */
+    assuranceDetailsRequired?: boolean;
```

Comment:
Can you please add the default value to the [GooglePay/defaultProps.ts](https://github.com/Adyen/adyen-web/blob/master/packages/lib/src/components/GooglePay/defaultProps.ts) file ?